frontend/src/global_styles/openproject/_variable_defaults.scss

```diff
@@ -148,4 +143,5 @@
   --list-item-hover--border-color: transparent;
   --list-item-hover--color: var(--accent-color);
   --link-text-decoration: none;
+  --task-board-header-background-color: #FFFFFF;
```

Comment:
Why should the task board header remain white?

Reply:
can we talk about task board colors with product team?